include/boost/compute/utility/program_cache.hpp

```diff
@@ -82,6 +82,25 @@ class program_cache : boost::noncopyable
         m_cache.clear();
     }
 
+    /// Clears persistent cache by deleting the boost_compute folder
+    void clear_persistent_cache()
```

Comment:
I think this could be a static method (so users can just call `program_cache::clear_persistent_cache()` directly without having to get/create a `program_cache` object).